draft-ietf-rats-corim.md

```diff
@@ -1816,6 +1864,13 @@ compared with the next entry of the Accepted Claims Set array.
 If all entries of the Reference Values array match a corresponding entry in the Accepted Claims Set array, then the `cryptokeys` Reference Value matches.
 Otherwise, `cryptokeys` does not match.
 
+##### Comparison for Integrity Registers {#sec-cmp-integrity-registers}
```

Comment:
For other types of measurement, the reference value chooses which values to compare. Can this comparison work the same way?
Diffs below suggesting what the text would look like if this suggestion is adopted.

Reply:
> For other types of measurement, the reference value chooses which values to compare. Can this comparison work the same way?

I guess the crucial point is whether we want the integrity registers to support "quote" semantics a la TPM, i.e., we allow evidence to be a user-defined subset of the complete state of the attester's IRs.  If so, evidence dictates the direction of matching.  If not, i.e., if we can assume that the quote is always _complete_, then we can flip the direction.

Reply:
The question is whether (a) the IR array should be considered an atomic measurement or (b) should each array member be atomic? If (b), then an IR as evidence could contain a superset of members contained in an RV array. The verifier should apply subset matching. If (a), then the matching semantic should be set equivalence. I don't feel strongly either way. If (a), then a second IR measurement can be used in evidence. However, if (b), the verifier would separate the IR into multiple IRs, and keep track of authority differences.

Reply:
Going back to our F2F discussion: *if* it can be assumed that a relying party cannot request a quote from the attester *independently* of the verifier - i.e., the RP cannot decide autonomously the geometry of the quote - then I am OK with @andrew-draper proposed text.

Reply:
@nedmsmith : We would await a new version of suggested `text` from your side and then we can proceed here!

Reply:
TL;DR the RP cannot decide autonomously the geometry of the quote and Andy's text is fine

I've been raising the question to the TCG IWG. There was a clear answer: the RP does not meddle with the geometry of the quote. The only usage scenario where that implicitly could be the case is where Attester owner and RP owner are collapsed roles and the local configuration of an Attester is required by certain RPs. In these cases the RP owner (which is also the Attester Owner) would also take on the role of Endorser and produce a CoRIM for the Verifier to take care of "the RP's needs". So, in the end the requirement would be again be addressed by the de-facto Endorser and not the RP.